.skills/testing_and_automation/android-emulator-skill/scripts/log_monitor.py

```
@@ -0,0 +1,70 @@
+            else:
+                print(f"Package {args.package} not running. Showing all logs.")
+        except Exception:
+            pass
+            
+    if args.grep:
+        full_cmd.extend(["|", "grep", args.grep])
+
+    print(f"Running: {' '.join(full_cmd)}")
+    try:
+        # Use subprocess directly to stream
+        process = subprocess.Popen(full_cmd, stdout=sys.stdout, stderr=sys.stderr)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)